doc/cla/corporate/innovyou.md

```diff
@@ -0,0 +1,15 @@
+Italy, 01/02/2024
+
+innoviú Srl agrees to the terms of the Odoo Corporate Contributor License
+Agreement v1.0.
+
+I declare that I am authorized and able to make this agreement and sign this
+declaration.
+
+Signed,
+
+Lorenzo Carta lorenzo_carta@innovyou.co https://github.com/LorenzoC0
```

Comment:
I believe you mean `lorenzo.carta@` (at least it's the email you are using)

Reply:
Yup, fixed on last push 👍🏻